README.md

```diff
@@ -169,23 +169,23 @@ If you are not that comfortable with the command line, here we provide an altern
 
 4. Click on the _Duplicate_ button. This will open the _Edit Ant Scenario_ window.
 
-5. Once in the _Edit Ant Scenario_, assign an appropriate name for the project (e.g., "MEI Mensural Schema - plica feature").
+5. Once in the _Edit Ant Scenario_, assign an appropriate name for the project (e.g., "MEI Mensural Schema - plica feature"). The storage option can be either _Project Options_ or _Global Options_.
 
 6. Change its `defaultSource` parameter by:
 
    a. Clicking on the _Parameters_ tab.
 
    b. Locating the `defaultSource` parameter and double-clicking on its value to change it. This will open the _Edit Parameter_ window.
 
-   c. Change the value of the `defaultSource` for the path of the MEI source file (`mei-source.xml`) found on your computer. You can do this by clicking on the folder icon to browse this file (it is located in your local copy of the music-encoding repo, in `music-encoding/source/mei-source.xml`) and opening it.
+   c. Change the value of the `defaultSource` for the path of the MEI source file (`mei-source.xml`) found on your computer. You can do this by clicking on the folder icon to browse this file (it is located in your local copy of the music-encoding repo, in `music-encoding/source/mei-source.xml`) and opening it. Make sure that the path start with *file* protocol `file:/` (e.g. `file:/D:/music-encoding/source/mei-source.xml`).
```

Comment:
```suggestion
   c. Change the value of the `defaultSource` for the path of the MEI source file (`mei-source.xml`) found on your computer. You can do this by clicking on the folder icon to browse this file (it is located in your local copy of the music-encoding repo, in `music-encoding/source/mei-source.xml`) and opening it. Make sure that the path starts with the *file* protocol, e.g., `file:/` (e.g. `file:/D:/music-encoding/source/mei-source.xml`).
```

Reply:
Does it need `file:/` if the user is on Mac or Linux?

Reply:
Thank you for your reply. It's not needed on mac, not sure about linux. Sorry forgot to mention the operation system.